Source/Extensions/Blazorise.ImageCropper/Enums/ViewMode.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Blazorise.ImageCropper;
+
+/// <summary>
+/// Define the view mode of the cropper. If you set viewMode to 0, the crop box can extend outside the canvas,
+/// while a value of 1, 2, or 3 will restrict the crop box to the size of the canvas. viewMode of 2 or 3 will
+/// additionally restrict the canvas to the container. There is no difference between 2 and 3 when the proportions
+/// of the canvas and the container are the same.
+/// </summary>
+public enum ViewMode
+{
+    /// <summary>
+    /// No restrictions
+    /// </summary>
+    Mode0 = 0,
+
+    /// <summary>
+    /// Restrict the crop box not to exceed the size of the canvas.
+    /// </summary>
+    Mode1 = 1,
```

Comment:
Instead of having Mode 1, 2, 3 maybe rename them to have a meaning?

- Mode0 to Default
- Mode1 to ExceedToCanvas
- Mode2 to FitContainer
- Mode3 to FillFitContainer

Reply:
was thinking the same but could not come up with good names yes